docs/source/explanation/algorithms/action/action_classification.rst

```diff
@@ -1,14 +1,47 @@
-Action Classificaiton
-=====================
+Action Recognition
+==================
+
+Action recognition is a problem of identifying the action that is being performed in a video. The input to the algorithm is a sequence of video frames, and the output is a label indicating the action that is being performed.
+
+For supervised learning we use the following algorithms components:
+
+- ``Augmentations``: We use standard data augmentations for videos, including random resizing and random cropping, horizontal flipping. We randomly sample a segment of frames from each video during training.
+
+- ``Optimizer``: We use the Adam with weight decay fix (AdamW) optimizer.
+
+- ``Learning rate schedule``: We use a step learning rate schedule, where the learning rate is reduced by a factor of 10 after a fixed number of epochs. We also use the Linear Warmup technique to gradually increase the learning rate at the beginning of training.

```

Comment:
I would suggest to highlight all the components with bold text. For example, **step learning rate schedule**, **Linear Warmup**, etc.